src/middlewared/middlewared/plugins/account.py

```diff
@@ -358,7 +366,7 @@ def validate_homedir_path(self, verrors, schema, data, users):
             verrors.add(f'{schema}.home', '"Home Directory" cannot contain colons (:).')
             return False
 
-        if data['home'] == DEFAULT_HOME_PATH:
+        if data['home'] in DEFAULT_HOME_PATHS:
```

Comment:
This looks wrong? `data['home']` has to be a literal `/var/empty` before this condition returns `True`. It's essentially the exact same operation before you made the change `==`